pkg/network/ovn_kubernetes.go

```diff
@@ -420,6 +420,7 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		productFlavor = "managed"
 		data.Data["CAConfigMap"] = bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.CAConfigMap
 		data.Data["CAConfigMapKey"] = bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.CAConfigMapKey
+		data.Data["PriorityClass"] = bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.PriorityClass
```

Comment:
It seems a bit odd to set this here rather than where `data.Data["HCPTolerations"]` is set but I don't think we get different results. I'll defer to Red Hat on the correct thing to do.

Reply:
Yeah, @bradbehle would you mind moving all 3 vars to the block that [sets the HCP vars from bootstrap](https://github.com/openshift/cluster-network-operator/blob/bb745971fceb366de50a75b3f373850af230fea1/pkg/network/ovn_kubernetes.go#L228)?